### PR TITLE
Expand description of about file

### PR DESCRIPTION
**Pull request by rien**

### CODE REVIEW

File: references/exercise-directory-structure/en.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ slug: references-exercise-directory-structure
 Inside an exercise repository, Dodona handles every directory containing a `config.json` file as a separate exercise. We expect this exercise directory to have a specific structure:
 
 - **A `config.json` file**: this file contains the [exercise-specific configuration](/references/exercise-config). This configuration will be merged with all `dirconfig.json` files in the exercise's ancestor directories. You can always override config values set by a higher directory.
-- **An optional `about.en.md` and/or `about.nl.md` file:** The content of these files will be shown on the exercise info page. These files are meant to give extra context to teachers who might be interested in using this exercise in a course.
+- **An optional `about.en.md` and/or `about.nl.md` file:** The content of these files will be shown on the exercise info page. These files are meant to give extra context to teachers who might be interested in using this exercise in a course. You may also use `about.md` and `readme.md`, but the localized alternatives take precedence over these files. Especially `readme.md` is useful because GitHub also displays this file when looking at the repository.
```
Comment (bmesuere) [inline references/exercise-directory-structure/en.md:19]: I would only mention the readme and not the about version.

**Comment by rien** [comment]
@bmesuere but readme doesn't support localization? Or would you want to completely get rid of `about.*` files?

**Comment by bmesuere** [comment]
@rien Hmm, I was under the impression that we would completely replace all mentions of "about" by "readme", but that we would keep about as a fallback in the code for backward compatibility?

**Comment by rien** [comment]
Aha, I thought we discussed that since only `readme.md` is rendered in GitHub, we would keep the current behavior so users have to explicitly choose which language they use as the default.

I don't mind changing this to allow and prefer `readme.#{lang}.md`, if you think that's better.

**Pull request merged**